creator/analyses/file_types.py

```diff
@@ -50,4 +50,19 @@
         ],
         "template": "family_trio_config.py",
     },
+    "GWO": {
+        "name": "Genomic Workflow Output Manifest",
+        "required_columns": [
+            'cavatica_id',
+            'cavatica_task',
+            'kf_biospecimen_id',
+            'kf_participate_id',
+            'kf_family_id',
+            'filepath',
+            'data_type',
+            'workflow_type',
+            'source_read'
+        ],
+        "template": "genomic_workflow_output_config.py",
```

Comment:
I think we need to make this template or else we may hit bugs when we try to return an extract config for this file type.

Reply:
Ok will add